index.js

```diff
@@ -65,45 +65,41 @@ module.exports = {
     const trueEmberVersion = emberVersion.match(/\d+\.\d+\.\d+/)[0];
 
     let options = {
-      envFlags: {
+      debugTools: {
+        isDebug: true,
         source: 'ember-compatibility-helpers',
-        flags: {
-          DEBUG: false
-        }
+        assertPredicateIndex: 1
       },
 
-      features: {
-        name: 'ember-compatibility-helpers',
-        source: 'ember-compatibility-helpers',
-        flags: {
-          HAS_UNDERSCORE_ACTIONS: !gte(trueEmberVersion, '2.0.0'),
-          HAS_MODERN_FACTORY_INJECTIONS: gte(trueEmberVersion, '2.13.0'),
-          HAS_DESCRIPTOR_TRAP: satisfies(trueEmberVersion, '~3.0.0'),
-          HAS_NATIVE_COMPUTED_GETTERS: gte(trueEmberVersion, '3.1.0-beta.1'),
-
-          IS_GLIMMER_2: gte(trueEmberVersion, '2.10.0'),
-
-          SUPPORTS_FACTORY_FOR: gte(trueEmberVersion, '2.12.0') || parentChecker.for('ember-factory-for-polyfill', 'npm').gte('1.0.0'),
-          SUPPORTS_GET_OWNER: gte(trueEmberVersion, '2.3.0') || parentChecker.for('ember-getowner-polyfill', 'npm').gte('1.1.0'),
-          SUPPORTS_SET_OWNER: gte(trueEmberVersion, '2.3.0'),
-          SUPPORTS_NEW_COMPUTED: gte(trueEmberVersion, '1.12.0-beta.1'),
-          SUPPORTS_INVERSE_BLOCK: gte(trueEmberVersion, '1.13.0'),
-          SUPPORTS_CLOSURE_ACTIONS: gte(trueEmberVersion, '1.13.0'),
-          SUPPORTS_UNIQ_BY_COMPUTED: gte(trueEmberVersion, '2.7.0')
+      flags: [
+        {
+          name: 'ember-compatibility-helpers',
+          source: 'ember-compatibility-helpers',
+          flags: {
+            HAS_UNDERSCORE_ACTIONS: !gte(trueEmberVersion, '2.0.0'),
+            HAS_MODERN_FACTORY_INJECTIONS: gte(trueEmberVersion, '2.13.0'),
+            HAS_DESCRIPTOR_TRAP: satisfies(trueEmberVersion, '~3.0.0'),
+            HAS_NATIVE_COMPUTED_GETTERS: gte(trueEmberVersion, '3.1.0-beta.1'),
+
+            IS_GLIMMER_2: gte(trueEmberVersion, '2.10.0'),
+
+            SUPPORTS_FACTORY_FOR: gte(trueEmberVersion, '2.12.0') || parentChecker.for('ember-factory-for-polyfill', 'npm').gte('1.0.0'),
+            SUPPORTS_GET_OWNER: gte(trueEmberVersion, '2.3.0') || parentChecker.for('ember-getowner-polyfill', 'npm').gte('1.1.0'),
+            SUPPORTS_SET_OWNER: gte(trueEmberVersion, '2.3.0'),
+            SUPPORTS_NEW_COMPUTED: gte(trueEmberVersion, '1.12.0-beta.1'),
+            SUPPORTS_INVERSE_BLOCK: gte(trueEmberVersion, '1.13.0'),
+            SUPPORTS_CLOSURE_ACTIONS: gte(trueEmberVersion, '1.13.0'),
+            SUPPORTS_UNIQ_BY_COMPUTED: gte(trueEmberVersion, '2.7.0')
+          }
         }
-      },
+      ],
 
       externalizeHelpers: {
         global: 'Ember'
-      },
-
-      debugTools: {
-        source: 'ember-compatibility-helpers',
-        assertPredicateIndex: 1
       }
     };
 
-    return [require.resolve('babel-plugin-debug-macros'), options];
+    return [require.resolve('babel-plugin-debug-macros'), options, 'ember-compatibility-helpers:debug-macros'];
```

Comment:
Need to register the plugin with an alternate name in Babel 7 because there's already another `debug-macros` plugin instance registered (apparently)